extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -478,12 +479,11 @@
 					item: _t.item,
 					value: value
 			}));
-			_t.suppressUpdate();
```

Comment:
[1] I understand that you need to obtain some properties from server so you don't want to suppress the callback entirely...